manifests/cinder-csi-plugin/cinder-csi-controllerplugin-rbac.yaml

```diff
@@ -122,6 +122,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots/status"]
     verbs: ["update"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
```

Comment:
@jichenjc with new snapshot-controller, the above some of the rbacs are not needed for the csi-snapshotter container, as the functionality have been moved to snapshot controller. Could you update the rbac here , https://github.com/kubernetes-csi/external-snapshotter/blob/v2.1.1/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml

Reply:
right, will give a try, thanks for the reminder